mkdocs.yml

```diff
@@ -1,10 +1,14 @@
 site_name: Kuadrant Documentation
 site_url: https://docs.kuadrant.io/
+repo_url: https://github.com/kuadrant
```

Comment:
Unsure where this is used, but I'd guess this should be https://github.com/kuadrant/docs.kuadrant.io ?

Reply:
It adds the link to GitHub on top right corner of the page.

<img width="444" alt="Screenshot 2023-07-05 at 11 53 27" src="https://github.com/Kuadrant/docs.kuadrant.io/assets/1842261/d94e47b1-a86a-48fa-a257-79d8fb53cd93">

Reply:
ah! in which case, the current link is fine I reckon. Ignore me ;)